src/NuGet.Clients/NuGet.PackageManagement.UI/PackageItemLoader.cs

```diff
@@ -276,6 +276,14 @@ public IEnumerable<PackageItemViewModel> GetCurrent()
                     }
                 }
 
+                var packageLevel = metadata.TransitiveOrigins != null ? PackageLevel.Transitive : PackageLevel.TopLevel;
+
+                var transitiveToolTipMessage = string.Empty;
+                if (packageLevel == PackageLevel.Transitive)
+                {
+                    transitiveToolTipMessage = metadata.Identity.Version + " by " + string.Join(", ", metadata.TransitiveOrigins);
```

Comment:
Use a string resource to generate tooltip message, example:


```suggestion
               {
                    transitiveToolTipMessage = string.Format(Resources.PackageVersionWithTransitiveOrigins, metadata.Identity.Version , string.Join(", ", metadata.TransitiveOrigins), CurrentThread.CurrentCulture);
```

Reply:
String concatenation like this will violate localization and make it not possible to translate the tooltip text.

Please put the entire string as a resource string and put placeholders (with {0}, {1}, etc.) and then use string.Format to format the string.